auth/auth-server/src/telemetry/sources/mod.rs

```diff
@@ -7,6 +7,11 @@ use renegade_api::http::external_match::AtomicMatchApiBundle;
 use renegade_circuit_types::{order::OrderSide, Amount};
 use renegade_common::types::token::Token;
 
+/// The gas estimation to use if fetching a gas estimation fails
+const DEFAULT_GAS_ESTIMATION: u64 = 4_000_000; // 4m
```

Comment:
just out of curiosity, where did this default come from, and can we document that?

Reply:
comes from https://github.com/renegade-fi/renegade/blob/main/workers/api-server/src/http/external_match.rs/#L62, documented this